plugin-server/src/main/ingestion-queues/analytics-events-ingestion-overflow-consumer.ts

```diff
@@ -49,3 +54,45 @@ export const startAnalyticsEventsIngestionOverflowConsumer = async ({
 
     return queue
 }
+
+export async function eachBatchIngestionFromOverflow(
+    payload: EachBatchPayload,
+    queue: IngestionConsumer
+): Promise<void> {
+    function groupIntoBatchesIngestion(kafkaMessages: KafkaMessage[], batchSize: number): KafkaMessage[][] {
```

Comment:
we don't need this - just use `eachBatch` from utils like we do in https://github.com/PostHog/posthog/blob/469057b905f7cdbe7d5143483f54e80377146461/plugin-server/src/main/ingestion-queues/batch-processing/each-batch-async-handlers.ts#L24

Reply:
We don't want to split by key here since this is the overflow topic we don't by definition guarantee single order processing (as the same team-distinct-id isn't in the same partition)

Reply:
I just used the function from the utils :+1: